api/src/main/java/org/openmrs/module/initializer/api/display/DisplaysCsvParser.java

```diff
@@ -0,0 +1,47 @@
+package org.openmrs.module.initializer.api.display;
+
+import static org.apache.commons.lang3.StringUtils.isBlank;
+
+import org.openmrs.OpenmrsObject;
+import org.openmrs.module.initializer.Domain;
+import org.openmrs.module.initializer.api.BaseLineProcessor;
+import org.openmrs.module.initializer.api.CsvLine;
+import org.openmrs.module.initializer.api.CsvParser;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class DisplaysCsvParser extends CsvParser<OpenmrsObject, BaseLineProcessor<OpenmrsObject>> {
+	
+	private CsvParser<? extends OpenmrsObject, ? extends BaseLineProcessor<? extends OpenmrsObject>> bootstrapParser;
+	
+	public void setBootstrapParser(
+	        CsvParser<? extends OpenmrsObject, ? extends BaseLineProcessor<? extends OpenmrsObject>> parser) {
+		this.bootstrapParser = parser;
+	}
+	
+	@Autowired
+	public DisplaysCsvParser(DisplayLineProcessor baseProcessor) {
+		super(baseProcessor);
+	}
+	
+	@Override
+	public Domain getDomain() {
+		return null;
+	}
```

Comment:
The parser doesn't provide a `Domain` to the loader. As a reminder, usually loaders infer from the parser-provided `Domain` all domain-related information that loaders are supposed to expose: domain name and domain order. This is specified at the loader level directly for this pre-loader.